TestAssets/TestProjects/TestAppSimple/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
Yeah at some point we need to bring those PortableTests into the main directory, now that everything is portable/standalone